into/backends/tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+@pytest.fixture(scope='session')
+def sc():
+    pyspark = pytest.importorskip('pyspark')
+    return pyspark.SparkContext('local', 'into-test')
+
+
+@pytest.fixture(scope='session')
+def sqlctx(sc):
+    sparksql = pytest.importorskip('pyspark.sql')
+    return sparksql.SQLContext(sc)
```

Comment:
I don't think this requires extra classes.  I think it means caching the SQLContext call in this function:

```python
+@convert.register(SchemaRDD, (RDD, PipelinedRDD))
+def rdd_to_schema_rdd(rdd, **kwargs):
+    return append(SQLContext(rdd.context), rdd, **kwargs)
```

Every time we move from pyspark to sparksql land we're making a new sqlcontext.  I'm suggesting something like the following

    SQLContext = memoize(SQLContext)